pkg/api/types.go

```diff
@@ -1865,3 +1863,39 @@ type ComponentStatusList struct {
 
 	Items []ComponentStatus `json:"items"`
 }
+
+// SecurityContext holds security configuration that will be applied to a container.  SecurityContext
```

Comment:
I don't think this comment is correct for the context, since in the internal API these fields are not on Container.  I think we can probably leave this comment out (except for the first sentence).